ibis/expr/builders.py

```diff
@@ -145,8 +146,8 @@ class WindowBuilder(Builder):
     how: Literal["rows", "range"] = "rows"
     start: Optional[RangeWindowBoundary] = None
     end: Optional[RangeWindowBoundary] = None
-    groupings: VarTuple[Union[str, Resolver, ops.Value]] = ()
-    orderings: VarTuple[Union[str, Resolver, ops.SortKey]] = ()
+    groupings: Selector | VarTuple[Union[str, Resolver, Selector, ops.Value]] = ()
```

Comment:
Is the first `Selector |` actually needed?

Reply:
Hm, maybe not!

Reply:
Nope, not needed!